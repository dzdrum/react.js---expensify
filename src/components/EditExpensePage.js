import React from 'react';

const EditExpensePage = (props) => {
    console.log(props);
    return (
        <div>
            Editing the expense of the id of {props.match.params.id}
        </div>
    );
}

export default EditExpensePage;