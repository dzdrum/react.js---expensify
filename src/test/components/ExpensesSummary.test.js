import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import React from 'react';

test('should render expensesSummary correctly with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={1094}/>);
  expect(wrapper).toMatchSnapshot();
});

test('should render expensesSummary correctly with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={23} expensesTotal={10943344}/>);
  expect(wrapper).toMatchSnapshot();
});
