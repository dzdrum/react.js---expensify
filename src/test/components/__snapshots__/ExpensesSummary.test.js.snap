// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensesSummary correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $10.94
  </h1>
</div>
`;

exports[`should render expensesSummary correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $109,433.44
  </h1>
</div>
`;
